core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -156,18 +194,28 @@ private int findFirstFreeDeviceIndex(User userEntity)
 	}
 
 	@Transactional
-	public void deleteDevice(User userEntity, UserDevice device)
+	public void deleteDeviceAndNotifyBuddies(UUID userId, UUID deviceId)
```

Comment:
🔧  The other methods also notify buddies, so we can simplify this to `deleteDevice` (or should add the `AndNotifyBuddies` to the other methods)

Reply:
Valid point. The normal behavior is to notify the buddies on such actions, so I've renamed the other public deletion method into ``deleteDeviceWithoutBuddyNotification``.